src/stdlib/system/system.bpf.c

```diff
@@ -0,0 +1,7 @@
+#define __VMLINUX_H__
+#include <vmlinux.h>
```

Comment:
You don't need `<vmlinux.h>`, you just need `<linux/types.h>` and a couple others. You should also `#define __KERNEL__` not `__VMLINUX_H__`

Note that you'll need to separate with newlines because clang-fmt wants to do weird header order that doesn't work for the kernel :)